src/api/che-login-manager.ts

```diff
@@ -185,7 +190,7 @@ export class CheServerLoginManager {
 
     const allLogins = new Map<string, string[]>()
     for (const [apiUrl, serverLogins] of Object.entries(this.loginData.logins!)) {
-      allLogins.set(apiUrl, Array.from(Object.keys(serverLogins)))
+      allLogins.set(apiUrl, [...Object.keys(serverLogins)])
```

Comment:
What is the purpose of this change?

Reply:
@mmorhun This change was forced by https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v6.0.1/docs/rules/prefer-spread.md... we can add a eslint comment if you wish